### ISSUE TITLE
dependency on multiworld even when not using it

### ISSUE THREAD

**Issue by dwiel** [vitchyr/rlkit#60]
I am using a custom environment, yet it seems that `HER` depends on `multiworld` anyway. The documentation says this shouldn't be necessary. Here is the stacktrace:
```
    from rlkit.samplers.data_collector import GoalConditionedPathCollector
  File "/root/src/sandbox/rlkit/rlkit/samplers/data_collector/__init__.py", line 6, in <module>
    from rlkit.samplers.data_collector.path_collector import (
  File "/root/src/sandbox/rlkit/rlkit/samplers/data_collector/path_collector.py", line 3, in <module>
    from rlkit.envs.vae_wrapper import VAEWrappedEnv
  File "/root/src/sandbox/rlkit/rlkit/envs/vae_wrapper.py", line 11, in <module>
    from multiworld.core.multitask_env import MultitaskEnv
ModuleNotFoundError: No module named 'multiworld.core'
```

**Comment by vitchyr** [comment]
Good point. Hmmm, I'm not sure if I want to remove the dependence on `MultitaskEnv` since that defines the interface. The `env_util` functions could also be copy/pasted from `multiworld`, though it seems like a bit redundant.

If you don't want to install `multiworld`, you can copy/paste the code from [here](https://github.com/vitchyr/multiworld/blob/master/multiworld/core/multitask_env.py), but otherwise I'm open to suggestions for how else you might fix this.

**Comment by dwiel** [comment]
Yeah that makes sense. Perhaps `multiworld` could be made pip installable
and included in the provided environments? It not so much that I dont want
to install, but that it looks like it's not installed by default in any
environments, and unless I'm mistaken is only installable via `pip install
git+https://github...`

**Comment by CarloLucibello** [comment]
`multiworld` is needed to run the DQN example on CartPole as well, which is not ideal:

```
$ python examples/dqn_and_double_dqn.py 
Traceback (most recent call last):
  File "examples/dqn_and_double_dqn.py", line 12, in <module>
    from rlkit.torch.dqn.dqn import DQNTrainer
  File "/home/carlo/Git/rlkit/rlkit/torch/dqn/dqn.py", line 10, in <module>
    from rlkit.torch.torch_rl_algorithm import TorchTrainer
  File "/home/carlo/Git/rlkit/rlkit/torch/torch_rl_algorithm.py", line 7, in <module>
    from rlkit.core.batch_rl_algorithm import BatchRLAlgorithm
  File "/home/carlo/Git/rlkit/rlkit/core/batch_rl_algorithm.py", line 4, in <module>
    from rlkit.core.rl_algorithm import BaseRLAlgorithm
  File "/home/carlo/Git/rlkit/rlkit/core/rl_algorithm.py", line 8, in <module>
    from rlkit.samplers.data_collector import DataCollector
  File "/home/carlo/Git/rlkit/rlkit/samplers/data_collector/__init__.py", line 6, in <module>
    from rlkit.samplers.data_collector.path_collector import (
  File "/home/carlo/Git/rlkit/rlkit/samplers/data_collector/path_collector.py", line 3, in <module>
    from rlkit.envs.vae_wrapper import VAEWrappedEnv
  File "/home/carlo/Git/rlkit/rlkit/envs/vae_wrapper.py", line 11, in <module>
    from multiworld.core.multitask_env import MultitaskEnv
ModuleNotFoundError: No module named 'multiworld'
```

**Comment by vitchyr** [comment]
Addressed with 60d1ab8416193fd462acd46ee37e797cdb20a195. Thanks for the feedback!
